org-transclusion.el

```diff
@@ -562,6 +568,35 @@ the rest of the buffer unchanged."
       (move-marker marker nil) ; point nowhere for GC
       list)))
 
+(defun org-transclusion-toggle ()
```

Comment:
What does this do that add/remove cannot do?

It looks to be that you would like to call `org-latex-preview`. Perhaps we could consider adding a hook within `org-transclusion-add` after the content has been inserted?

Reply:
I don't think toggle adds anything new other than add/remove, but it's the main way in which I ended up using transclusion: as a way to preview content quickly. I think something like this should at least be natively there, even if it doesn't necessarily "add" a functionality.

The hook makes sense, as latex previews get broken a lot when I toggle links often.

Reply:
I've implemented the hook and created a new feature branch for the hook. 
Have a look at PR #124.

I don't use `latex` etc. and can't get the latex preview working, so I'd appreciate it if you could test it on your end.